federation-js/src/__tests__/testSetup.ts

```diff
@@ -0,0 +1 @@
+import 'apollo-federation-integration-testsuite';
```

Comment:
Similar comment here, can we add a comment saying this loads the matchers to make it more obvious?

Reply:
Yeah, I agree.  While _most_ `import 'module'` usage _has_ side-effects, I prefer annotating them with what those side-effects are.

Reply:
I changed the imports to `import 'apollo-federation-integration-testsuite/dist/matchers';` to be explicit about this.